services/src/api/model/datatypes.rs

```diff
@@ -1953,9 +2046,97 @@ impl From<MultiLineString> for geoengine_datatypes::primitives::MultiLineString
     }
 }
 
+impl ToSql for MultiLineString {
```

Comment:
why do you not use: postgres_types::Type::PATH::LINE_ARRAY ?

Reply:
1. We need multiple line strings and not lines.

2. 
> Lines are represented by the linear equation Ax + By + C = 0, where A and B are not both zero.

I thought we wouldn't want that.